### ISSUE TITLE
have you seen

### ISSUE THREAD

**Issue by marcus7777** [polymer-spain/polymerIDE#1]
http://open-elements.org/bower_components/pair-production/demo.html

**Comment by lruizr** [comment]
Hi! We have seen the demo, and we are also working in this field of web
components. If you are interested in, maybe we can collaborate.

Thanks in advance for your atention,

Regards,

Luis.

2015-09-15 8:45 GMT+02:00 Marcus Hitchins notifications@github.com:

**Comment by marcus7777** [comment]
Yes I'd love to collaborate anything that makes my life of making elements easier would be awesome as the polymer team Might Say.
